amlb/runners/singularity.py

```diff
@@ -85,13 +85,11 @@ def _start_container(self, script_params=""):
         for d in [in_dir, out_dir, custom_dir]:
             touch(d, as_dir=True)
         script_extra_params = "--session="  # in combination with `self.output_dirs.session` usage below to prevent creation of 2 sessions locally
-        inst_name = self.sid
```

Comment:
@franchuterivera I removed some variables that didn't seem to be used. From `singularity run` docs I also didn't see a way to set a name for a container, so I think this is likely just left over from a copy-paste. Do you think the modifications I made here are correct?

Reply:
@eddiebergman I am not sure if Francisco is still reachable through Github, but since you likely also use Singularity perhaps you can chime in. It would be much appreciated.

Reply:
I unfortunatly don't use singularity, perhaps @neeratyoy or @mfeurer can chime in?

Reply:
to reviewers: you can ignore this conversation if you're not familiar.

Reply:
hope not too late
I do work with Singularity but will not claim I understand the whole setup completely
though I have mainly been building containers and already had code to pull and run it
in our current project, we pull the container using its name and source 
and subsequently _start_ the singularity instance by name [here](https://github.com/automl/HPOBench/blob/master/hpobench/container/client_abstract_benchmark.py#L298)

Reply:
Thanks for having a look! The question here specifically is whether the `inst_name` is likely a left-over from copy-pasting docker code and would have no function for a `singularity run` command. It's currently not included in the command, and I can not see a way in the singularity docs to set a "name" for an instance. (not sure if using the right terminology for singularity, but the _image_ from which a container is made is still specified, of course).